### PR TITLE
206 Fix telegram notifications

### LINKED ISSUE #206: Fix Telegram bot failing to send messages to channels
**Describe the bug**
The Telegram bot failed to send a couple of updates to the Dota 2 feeder channel, while the Discord bot correctly notied its subscribers.
This might be connected to the bot hitting the API limits or failing to get the channel permissions.
The channel permissions should probably only be checked when performing a user command, not every time a message is sent. In those situation it's probably enough to catch and handle errors accordingly.

**Expected behavior**
All channels should be properly notified. If the API limit is hit, the notifications should be sent out with a small delay.

### PR DESCRIPTION
**Pull request by TimJentzsch**
**Description**:
Removes the permission checks when sending messages to Telegram channels. Hopefully this will fix some channels not recieving notifications or the bot hitting the rate limit.
Additionally, this adds channel labeling. Bot owners can use `label <bot name> <channel id> <channel label>` to add a label to a channel. This will be used during logging to simplify debugging.
Closes #206.

**Checklist**:

- [x] Tested the bot on both clients with a few commands
- [x] Updated the [`README`](README.md) if necessary
- [x] Updated the version string in the [`package.json`](package.json) if necessary

### CODE REVIEW

File: src/commands/commands.ts [hunk 1]
```diff
@@ -685,6 +685,163 @@ const debugCmd = new SimpleAction(
...
+    } else {
+      // No bot specified, take the bot that the message was written on
+      channelBot = bot;
+    }
+
+    let labelChannel = message.channel;
+
+    if (channelID === 'this') {
+      // Take this channel and this bot
+      labelChannel = message.channel;
```
Comment (zachkont) [inline src/commands/commands.ts]: very minor but this line seems unnecessary

File: src/commands/commands.ts [hunk 2]
```diff
@@ -685,6 +685,163 @@ const debugCmd = new SimpleAction(
...
+      // Update label
+      existingChannel.label = label;
+
+      // Remove unnecessary entries
+      if (
+        existingChannel.gameSubs.length === 0 &&
+        !existingChannel.prefix &&
+        !existingChannel.label
+      ) {
+        channelBot.logger.debug('Removing unnecessary channel entry...');
```
Comment (zachkont) [inline src/commands/commands.ts]: This log could also use a channel label/id

File: src/commands/commands.ts (hunk 1 above)
Comment (TimJentzsch) [reply src/commands/commands.ts]: Removed.

File: src/commands/commands.ts (hunk 2 above)
Comment (TimJentzsch) [reply src/commands/commands.ts]: Added.

**Pull request merged**